scripts/ci/jobs/ocp_sensor_integration_tests.py

```diff
@@ -15,11 +15,5 @@
 os.environ["ROX_ACTIVE_VULN_MGMT"] = "true"
 
 ClusterTestRunner(
-    pre_test=PreSystemTests(),
```

Comment:
I'd consider keeping this and adding it to the `gke_` version. It records useful info about the cluster under test which can help with debugging. If you don't need images you will need to add a flag to skip the poll https://github.com/stackrox/stackrox/blob/master/.openshift-ci/pre_tests.py#L32

For the post stuff it makes sense to remove it until you get a better idea about what you want to save to help debug failures.

Reply:
Enabled pre-test and added a bool param to control the polling in [046cd57](https://github.com/stackrox/stackrox/pull/6484/commits/046cd57d43664c8d660042e1d2ba0dfa21dc83b2)

Reply:
still missing in scripts/ci/jobs/ocp_sensor_integration_tests.py